Preface/Preface.pillar

```diff
@@ -58,12 +57,13 @@ available at *http://www.deepintopharo.org*
 
 Do not be frustrated by parts of Pharo that you do not immediately understand.
 You do not have to know everything! Alan Knight expresses this as follows:
-
+{{{latex:\begin{quote} }}}
```

Comment:
@DamienCassou - I would actually argue with the recommendation of ``@@note`` instead of a LaTeX quote tag (``{{{latex:\begin{quote} }}}``).

Pillar ``@@note`` prepends **Notes:** to the section, when rendering it in PDF. There are two problems with this: 1) It says Notes plural. Which is incorrect semantically -- the line is a single Note. 2) More importantly, this is NOT a note, but an actual quote from an author. And so should be formatted differently.